actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -134,13 +134,15 @@ def select_tag(name, option_tags = nil, options = {})
 
         if options.include?(:include_blank)
           include_blank = options.delete(:include_blank)
+          options_for_blank_options_tag = { value: '' }
 
           if include_blank == true
             include_blank = ''
+            options_for_blank_options_tag[:label] = ' '
```

Comment:
No that is invalid as well.

On Wednesday 11 May 2016, Jeremy Daer <notifications@github.com> wrote:

> In actionview/lib/action_view/helpers/form_tag_helper.rb
> <https://github.com/rails/rails/pull/24923#discussion_r62746660>:
>
> >
> >            if include_blank == true
> >              include_blank = ''
> > +            options_for_blank_options_tag[:label] = ' '
>
> Does label="" (empty string) work? Or is it treated as an absent label
> attr?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/24923/files/22b731188645d2b2c70725caad2bbe20411c05d8#r62746660>
>


-- 
Vipul A.M.
+91-8149-204995